configure.ac

```diff
@@ -78,6 +78,25 @@ fi
 
 AM_CONDITIONAL(HAVE_SYSTEMD, [test "$have_systemd" = "yes"])
 
+AC_ARG_WITH(elogind,
+            AS_HELP_STRING([--with-elogind],
+            [Use libelogind instead of libsystemd-login]),
+            [with_elogind=$withval], [with_elogind=no])
+
+use_elogind=no
+if test "x$with_elogind" = "xyes"; then
+    PKG_CHECK_MODULES(LIBELOGIND,[libelogind], [use_elogind=yes], [use_elogind=no])
```

Comment:
AFAICT `LIBELOGIND_LIBS` and `_CFLAGS` are not used, so how is the linking supposed to work?  The simplest solution would be to use `SYSTEMD` here as well, as it's an alternate for it.

Reply:
`--with-elogind` does *NOT* fail if elogind is not found (as you have `use_elogind=no` in the not-found path, even for the `--with-elogind` case)

Reply:
I added similar logic to that of systemd,